linux_os/guide/system/accounts/accounts-restrictions/root_logins/use_pam_wheel_for_su/rule.yml

```diff
@@ -45,8 +45,12 @@ fixtext: |-
 
     In file "/etc/pam.d/su", uncomment the following line:
 
-    "#auth    required    pam_wheel.so use_id"
+    "#auth    required    pam_wheel.so use_uid"
 
     $ sed '/^[[:space:]]*#[[:space:]]*auth[[:space:]]\+required[[:space:]]\+pam_wheel\.so[[:space:]]\+use_uid$/s/^[[:space:]]*#//' -i /etc/pam.d/su
```

Comment:
You should add :
If necessary, create a "wheel" group and add administrative users to the group.